File: testers/grpc/java-quarkus/generated-and-checked-in/com/example/grpc/CustomerId.java
package com.example.grpc;



/** Wrapper type for java.lang.String */
public record CustomerId(String value) {
  public CustomerId withValue(String value) {
    return new CustomerId(value);
  }

  @Override
  public java.lang.String toString() {
    return value.toString();
  }

  /** Create a CustomerId from a raw value */
  static public CustomerId valueOf(String v) {
    return new CustomerId(v);
  }

  /** Get the underlying value */
  public String unwrap() {
    return this.value();
  }
}

File: testers/grpc/scala-cats/generated-and-checked-in/com/example/grpc/CustomerId.scala
package com.example.grpc



/** Wrapper type for java.lang.String */
case class CustomerId(value: String) extends scala.AnyVal {
  /** Get the underlying value */
  def unwrap: String = {
    return this.value
  }
}

object CustomerId {
  /** Create a CustomerId from a raw value */
  def valueOf(v: String): CustomerId = {
    return new CustomerId(v)
  }
}